test/system/libs/helpers.bash

```diff
@@ -286,6 +286,36 @@ function pull_default_image() {
 }
 
 
+# Copies the system's default image to Podman's image store
+# and create a copy with a "-copy" suffix in a human readable name
+#
+# See pull_default_image() for more info.
+# Parameters:
+# ===========
+# - distro - os-release field ID (e.g., fedora, rhel)
+# - version - os-release field VERSION_ID (e.g., 33, 34, 8.4)
+function pull_and_rename_default_image() {
+  local distro
+  local version
+  local image
+  
+  distro="$1"
+  version="$2"
+  image="${IMAGES[$distro]}:${version}"
```

Comment:
This could be simpler if we just used the default image for the host where the tests are running.